SPECS/openvswitch/CVE-2021-3905.nopatch

```diff
@@ -0,0 +1,3 @@
+CVE-2022-4515 - this does not apply to version 2.15.7, which already has the following patch:
```

Comment:
Is it possible that NIST will just update their entry and the CVE would be no-op for us?

Reply:
Yes, I think so, but this will clear the CVE from our dashboard sooner. I can also reach out to them to see if they can update the entry.